spec/unit/actions/synchronize_clients_spec.rb

```diff
@@ -23,6 +24,20 @@
       }
     }
   end
+  let(:sync_clients_detail_result) do
+    {
+      "items" => [
+        {
+          "status" => "error",
+          "id" => "pid",
+          "error" => {
+            "message" => "Error sync ds",
+            "parameters" => "test_ds"
```

Comment:
I should use meaningful name instead of the bad name (test1, test2, test_ds,....)
expected_error_ds or snowflake_ds or expected_success_ds or expected_warning_ds

```suggestion
            "parameters" => "expected_error_sync_ds"
```

Reply:
Done update code